### DISCUSSION TITLE
renderForms was removed in 21.0.0-alpha.14

### DISCUSSION THREAD

**Discussion by Julian-B90** [stephanrauh/ngx-extended-pdf-viewer#2483] [General]
Hey why renderForms was removed? In this  Commit https://github.com/stephanrauh/ngx-extended-pdf-viewer/commit/5898f51eebd9344c16a9b8a5f2cf8963ae08ff63 ?

**Comment by stephanrauh** [comment]
I've removed it because the pdf.js team has removed the flag from their library. In other words, it was a flag that didn't influence anything.

The flag had been introduced when form support was implemented. The idea was to activate it on developer and QA machines without publishing it officially until it was thoroughly tested. It seem the test phase is over, so form support is always active.

**Reply by Julian-B90** [reply]
Thank you very much for your answer and your great work on this library. ❤️❤️
